framework/views/_dropColumns.php

```diff
@@ -5,6 +5,6 @@
     'foreignKeys' => $foreignKeys,
 ]);
 
-foreach ($fields as $field): ?>
+foreach ($fields as $field) : ?>
```

Comment:
This seems to be correct: https://github.com/squizlabs/PHP_CodeSniffer/issues/366#issuecomment-66937317

> I'm not sure what PSR2 actually intends in this case. They don't mention the alternate syntax anywhere. So I still need to enforce the spaces after opening parentheses error, which means you code will look like this after I fix these issues: